android/app/src/main/java/com/coboltforge/dontmind/multivnc/ConnectionBean.kt

```diff
@@ -0,0 +1,151 @@
+package com.coboltforge.dontmind.multivnc
+
+import android.os.Parcelable
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+@Entity(tableName = "CONNECTION_BEAN")
+data class ConnectionBean(
+        @JvmField
+        @PrimaryKey(autoGenerate = true)
+        @ColumnInfo(name = "_id")
+        var id: Long = 0,
+
+        @JvmField
+        @ColumnInfo(name = "NICKNAME")
+        var nickname: String? = "",
```

Comment:
Why not `String`?

Reply:
Because column `NICKNAME` is nullable in database. If we use `String` here, Room throws an exception because schema doesn't match.

Reply:
Alright.